components/develop/modules/ROOT/pages/sfuel/distribution/proof-of-work-distribution.adoc

```diff
@@ -0,0 +1,46 @@
+= Proof of Work Distribution
+
+xref:../../proof-of-work.adoc[Proof of Work] distribution is the recommended way for applications to distribute sFUEL to their users.
+This distribution method takes advantage of the PoW feature to allow users to onboard in a decentralized and frictionless manner.
+
+[Note]
+The Proof of Work distribution system relies on an on-chain faucet. If this faucet runs out of sFUEL, users will not be able to attain sFUEL on your chain.
+
+== The Smart Contract
+
+The smart contract portion of this distribution method can be very simple or as complex as needed to faciliate distribution.
+
+The only requirement is that a function must pay out to an address provided as input. Additionally, the more complex logic in the payout function, the longer
+the proof of work will have to run on the client to generate a big enough number.
+
+Additionally, it is recommended that the contract has limits in place so that one address cannot keep filling itself up
+
+
+== Distribution on a Dedicated SKALE Chain
+
+Distributing sFUEL from an onchain contract on your own dedicated chain requires the deployment of a distribution contract.
+
+An example of this contract can be found xref:../../proof-of-work.adoc#_pow_browser_usage_algorithm[here in the docs].
+
+You can also use the link:https://github.com/Dirt-Road-Development/sfuel-distribution-contract[community build sFUEL Distribution Contract] which is being used on Calypso, CryptoBlades, Chaos Testnet, Human Protocol Chain, Tank Chain, and more! 
+
+[NOTE]
+The community contract above is based on the concepts from the SKALE provided example.
+
+[NOTE]
+The Europa Liqudity Hub also uses a variation of this contract.
+
+== Distribution on a Hub Chain
+
+Deploying onto a SKALE Hub? Utilize one of the pre-existing sFUEL Distribution contracts. Don't see your chain? Head over to the link:https://discord.gg/skale[SKALE Discord] and request it!
+
+[width="100%"]
+=== Available On-Chain Faucets
+|===
+| Chain             | Mainnet Address                               | Testnet Address                               | Function Signature    |
```

Comment:
apply header rows https://docs.asciidoctor.org/asciidoc/latest/tables/add-header-row/#explicitly-assign-header-to-the-first-row

Reply:
Resolved by 587b363342cedc0cfda5f502b4ba0be8836f1159